app/src/main/java/com/framgia/moneymanagement/screen/statistics/StatisticsAdapter.java

```diff
@@ -0,0 +1,51 @@
+package com.framgia.moneymanagement.screen.statistics;
+
+import android.content.Context;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+
+import com.framgia.moneymanagement.R;
+
+public class StatisticsAdapter extends FragmentPagerAdapter {
+    private Context mContext;
+    private static final int NUMBER_ZEZO = 0;
+    private static final int NUMBER_ONE = 1;
+    private static final int NUMBER_TWO = 2;
+
+    public StatisticsAdapter(FragmentManager fm, Context context) {
+        super(fm);
+        mContext = context;
+    }
+
+    @Override
+    public Fragment getItem(int position) {
+        switch (position) {
+            case NUMBER_ZEZO:
+                return StatisticsLastMonthFragment.newInstance();
+            case NUMBER_ONE:
+                return StatisticsThisMonthFragment.newInstance();
+            default:
+                return StatisticsThisMonthFragment.newInstance();
+        }
+    }
+
+    @Override
+    public int getCount() {
+        return NUMBER_TWO;
+    }
+
+    @Nullable
+    @Override
+    public CharSequence getPageTitle(int position) {
+        switch (position) {
+            case NUMBER_ZEZO:
+                return mContext.getResources().getString(R.string.statistics_last_month);
+            case NUMBER_ONE:
```

Comment:
duplicate dòng 46 và 48
```suggestion
            default:
            case NUMBER_ONE:
```